src/components/pages/index.js

```diff
@@ -20,6 +21,12 @@ const HomePage = () => {
         src="/images/Real-Dev-Squad@1x.png"
         alt="real-dev squad"
       />
+      {dev && (
+        <>
```

Comment:
Why didn't we go with React.Fragment here?

Reply:
Either way, it's fine 😅 
I personally don't like using `React.fragment` when we have the shorthand available.